lib/fluent/plugin/in_tail.rb

```diff
@@ -384,6 +398,9 @@ def close_watcher_handles
 
     # refresh_watchers calls @tails.keys so we don't use stop_watcher -> start_watcher sequence for safety.
     def update_watcher(path, pe)
+      log_msg = "detected rotation of #{path}"
+      log_msg << "; waiting #{@rotate_wait} seconds" # wait rotate_time if previous file exists
+
```

Comment:
Missing `log.info` here?

Reply:
Good catch…thank you 🙏  https://github.com/fluent/fluentd/pull/2832/commits/994086913afb32f6697d8960c840a9458e63009b